### ISSUE TITLE
Linux: Fatal error on second run

### ISSUE THREAD

**Issue by ohthehugemanatee** [eul-im/eul#5]
Starting eul briefly shows me my slack channels, and then...

```
2017/07/27 09:56:52 Starting eul v0.16...
fatal error: concurrent map read and map write
```

**Comment by voltapp2** [comment]
Hi @ohthehugemanatee,

This was fixed 2 days ago for Windows and Mac, and I forgot to do this for Linux. The new 0.17 release will take care of that. It should be live in a couple of hours.

**Comment by ohthehugemanatee** [comment]
Thanks for your quick feedback! Looking forward to it. :)

**Comment by voltapp2** [comment]
@ohthehugemanatee Sorry for the delay, this has been fixed in 0.17. Can you please try again?

**Comment by ohthehugemanatee** [comment]
confirmed fixed.
